Jellyfin.Api/Controllers/LibraryStructureController.cs

```diff
@@ -75,7 +75,9 @@ public ActionResult<IEnumerable<VirtualFolderInfo>> GetVirtualFolders()
     [HttpPost]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     public async Task<ActionResult> AddVirtualFolder(
-        [FromQuery] string name,
+        [FromQuery]
+        [RegularExpression(@"^(?:\S(?:.*\S)?)$", ErrorMessage = "Library name cannot be empty or have leading/trailing spaces.")]
```

Comment:
Can this be simplified to `^\b.+\b$`?

Reply:
Would't this also reject some valid library names that has punctuation marks at the beginning or the end of the name? Like `My Favourites!` will fail this regex